pkg/cmd/repo/delete/http.go

```diff
@@ -10,6 +10,10 @@ import (
 )
 
 func deleteRepo(client *http.Client, repo ghrepo.Interface) error {
+	client.CheckRedirect = func(req *http.Request, via []*http.Request) error {
```

Comment:
This mutates the client. Could we create a new one via shallow clone and then edit the CheckRedirect field?